idaes/models/properties/general_helmholtz/helmholtz_functions.py

```diff
@@ -154,12 +154,165 @@ class AmountBasis(enum.Enum):
         "arg_units": [dimensionless, dimensionless, dimensionless],
         "doc": "cp(comp, delta, tau)",
     },
-    "w_func": {  # entropy
+    "w_func": {  # speed of sound
         "fname": "w",
         "units": pyo.units.m / pyo.units.s,
         "arg_units": [dimensionless, dimensionless, dimensionless],
         "doc": "w(comp, delta, tau)",
     },
+    "v_func": {  # specific volume
+        "fname": "v",
+        "units": pyo.units.m**3 / pyo.units.kg,
+        "arg_units": [dimensionless, pyo.units.kg / pyo.units.kg, pyo.units.kPa],
```

Comment:
Delta and Tau are some normalized (and possibly inverted?) versions of density and temperature, respectively, right? Assuming "arg_units" means, "units with which the function is called", we've got an issue here.

Reply:
Yeah, those units are wrong.  I don't actually use that function anywhere yet so it did cause a problem.  Will fix.